quality_indicators/indicator_schema.json

```diff
@@ -0,0 +1,241 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://everse.software/indicator_schema.json",
+    "type": "object",
+    "default": {},
+    "title": "EVERSE indicator schema",
+    "required": [
+        "identifier",
+        "name",
+        "description",
+        "status",
+        "quality-dimension",
+        "release-date",
+        "doi",
```

Comment:
doi seems redundant with identifier. As discussed today, it may have been legacy from the old schema

Reply:
done